includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1762,7 +1769,12 @@ public static function get_reader_friendly_error_type_text( $error_type ) {
 	 * @return string The label.
 	 */
 	public static function get_details_summary_label( $validation_error ) {
-		if ( self::INVALID_ATTRIBUTE_CODE === $validation_error['code'] || self::INVALID_ELEMENT_CODE === $validation_error['code'] ) {
+		if (
+			self::INVALID_ATTRIBUTE_CODE === $validation_error['code'] ||
+			self::INVALID_ELEMENT_CODE === $validation_error['code'] ||
+			'invalid_processing_instruction' === $validation_error['code'] ||
```

Comment:
Maybe `'invalid_processing_instruction'` could be in a class constant like `self::INVALID_PROCESSING_INSTRUCTION_CODE`, similar to some other codes above.

Reply:
Yes, I thought about that, but I want to wait to add more constants until we decide what to do with aligning error codes with the AMP Validator (#1420).

Reply:
Sure, that sounds good.